pkgs/shells/fish/plugins/colored-man-pages.nix

```diff
@@ -0,0 +1,19 @@
+{ lib, buildFishPlugin, fetchFromGitHub }:
+
+buildFishPlugin rec {
+  pname = "colored-man-pages";
+  version = "unstable-2022-04-30";
+
+  src = fetchFromGitHub {
+    owner = "patrickf1";
+    repo = "colored_man_pages.fish";
+    rev = version;
```

Comment:
Doesn't build:

```
~/src/nixpkgs add-color-man-pages ≡
❯ nix build --impure --expr 'with import ./. {}; wrapFish {
     pluginPkgs = [ pkgs.fishPlugins.colored-man-pages ];
   }'
error: builder for '/nix/store/wc2pzlcp5hv9b31zwrzfzhkg3l8srxvg-source.drv' failed with exit code 1;
       last 8 log lines:
       >
       > trying https://github.com/patrickf1/colored_man_pages.fish/archive/unstable-2022-04-30.tar.gz
       >   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
       >                                  Dload  Upload   Total   Spent    Left  Speed
       >   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
       >   0    14    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
       > curl: (22) The requested URL returned error: 404
       > error: cannot download source from any mirror
       For full logs, run 'nix log /nix/store/wc2pzlcp5hv9b31zwrzfzhkg3l8srxvg-source.drv'.
error: 1 dependencies of derivation '/nix/store/68rr008m1slx4245qzll2219jmx7b766-fishplugin-colored-man-pages-unstable-2022-04-30.drv' failed to build
error: 1 dependencies of derivation '/nix/store/8g389r5sng65s2clx74hjk3js2kxg8hs-fish.drv' failed to build
```

Since there's no tagged release, we might need to use the commit hash matching the date:

```suggestion
    rev = "f885c2507128b70d6c41b043070a8f399988bc7a";
```

Reply:
Ah sorry I did a last minute modification to match the version name as in other fish plugins and forgot to change it there. I did not notice it on my end as the package path was already in my nix store and Nix did not try to re download it.

It's now fixed thanks!